Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
+      - >
```

Comment:
The `dev_server` task has a long command. Consider breaking it into multiple lines for better readability. Each flag can be on a new line with a leading `
` or use the YAML block scalar `|` for clearer formatting.